File: NumbersAndFormatting.java
public class NumbersAndFormatting {
   public static void main(String[] args) {
   int x = 4;
   int y = 2;
   double radius = 3.7;
   double pi = 3.14;
   System.out.println("x+y=" + (x + y));
   System.out.printf("y+radius=%.2f\n", (y + radius));
   System.out.println("x/y=" + (x/y));
   System.out.println("x%y=" + (x%y));
   System.out.println("y%x=" + (y%x));
   System.out.printf("radius/x=%.2f\n", (radius/x));
   System.out.printf("The area of a circle with radius %.2f is %.2f", radius, (pi*(radius*radius)));
   }
}

File: NumbersAndFormatting.py
x = 4
y = 2
z = x + y
radius = 3.7
pi = 3.14
print("x+y=" + str(x+y))
print("y+radius=" + ('%.2f' % (y+radius)))
print("x/y=" + str(int(x/y)))
print("x%y=" + str(x%y))
print("y%x=" + str(y%x))
print("radius/x=" + str(round(radius/x, 2)))
print("The area of a circle with radius " + str('%.2f' % radius) + " is " + str('%.2f' % (pi*(radius*radius))))